fighters/lucas/src/acmd/throws.rs

```diff
@@ -10,22 +10,39 @@ unsafe fn game_catch(fighter: &mut L2CAgentBase) {
         ArticleModule::change_motion(fighter.module_accessor, *FIGHTER_LUCAS_GENERATE_ARTICLE_HIMOHEBI, Hash40::new("catch"), false, 0.0);
         //FT_MOTION_RATE(fighter, 48.0/(69.0-1.0));
     }
+    frame(lua_state, 8.0);
+    if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 0.5);
+    }
     frame(lua_state, 9.0);
     if is_excute(fighter) {
+        FT_MOTION_RATE(fighter, 1.0);
```

Comment:
convert motion rating to standard format and be sure not to wrap them in `is_excute`